fuel/schemes.py

```diff
@@ -232,3 +232,48 @@ def get_request_iterator(self):
         indices = list(self.indices)
         self.rng.shuffle(indices)
         return iter_(indices)
+
+
+def k_fold_cross_validation(scheme_class, num_examples, k, strict=True,
+                            **kwargs):
+    """Return pairs of schemes to be used for cross-validation.
+
+    Parameters
+    ----------
+    scheme_class : class
```

Comment:
I'd be more specific here and specify that it's either an `IndexScheme` instance or a `BatchScheme` instance.